builtin/providers/aws/resource_aws_route.go

```diff
@@ -320,6 +326,11 @@ func findResourceRoute(conn *ec2.EC2, rtbid string, cidr string) (*ec2.Route, er
 		return nil, err
 	}
 
+	if len(res.RouteTables) < 1 || res.RouteTables[0] == nil {
```

Comment:
The response is called `resp` here, not `res`. I think we should rename it `resp` in the other method, honestly. `resp` is more telling to me than `res`...